src/utils.js

```diff
@@ -60,3 +60,5 @@ export function getUpcomingDate(dateTimeList) {
 }
 
 export const isAdmin = (user) => user?.admin === true;
+export const isVisitor = (user) => user === 'visitor';
```

Comment:
@ranyitz `isAdmin` and `isAuthenticated` receive actual user as an argument, but `isVisitor` gets the role?
can you make `isVisitor` get the user instance as well?

Reply:
Oh, I see it does get the user. so shouldn't we check some property on the user?

Reply:
I did this, because I wanted to have a different state for visitor, we used to put `null` and it made it difficult to distinguish between visitor, user, and a state when we still don't know.

We can refactor that later (to have the same shape)